index.html

```diff
@@ -2810,6 +2813,84 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section id="solid-color-icons">
+        <h2>
+          Solid color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a single color
+          or gradient, where only the transparency of the icon can be
+          controlled. As web applications should work across multiple
+          platforms, it it possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>monochrome</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>monochrome</a> icon, the user agent MUST NOT use
+          the red component, green component, or blue component of a pixel. If
+          it has alpha equal to zero, the user agent SHOULD NOT display it.
+          If it the alpha component is greater than zero, the user agent SHOULD
+          display it with any tint.
+        </p>
+        <p>
+          If no monochrome icon is defined when an icon with a solid icon must
+          be presented, the user agent MAY show a fallback icon.
+        </p>
+        <p class="note">
+          Designers of <a>monochrome</a> icons are recommended to set all
+          pixels to black and use transparency to create a silhouette of their
+          icon.
+        </p>
+        <p>
+          The user agent MAY enlarge the icon by adding additional padding.
+        </p>
+        <p>
+          The user agent MAY add a background of any color behind transparent
+          pixels.
+        </p>
+        <section>
+          <h3>
+            Example usage of solid color icons
+          </h3>
+          <h2 class="icon-title">
+            Usage examples
+          </h2>
+          <div class="icons">
+            <div>
```

Comment:
should this be a `<figure>`?

Reply:
I'm not sure, I based this on the maskable icon examples present in the spec.

Reply:
Using `<figure>` messes with the styling since extra properties are injected, so I'll leave this as `<div>` for now